src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -40,6 +44,12 @@
         };
     }
 
+    private static Project[] getSampleProjects() {
+        return new Project[]{
```

Comment:
Missing space

Reply:
yeap surprised my checkstyle didn't catch that. thanks!